projects/addon-doc/components/documentation/documentation-property-connector.directive.ts

```diff
@@ -108,6 +110,14 @@ export class TuiDocDocumentationPropertyConnectorDirective<T>
         console.info(this.attrName, event);
 
         this.emits$.next(this.emits$.value + 1);
+
+        let content: string | undefined;
+
+        if (event !== undefined) {
+            content = tuiInspectAny(event, 2);
+        }
+
+        this.alerts.open(content, {label: this.attrName}).subscribe();
```

Comment:
Can we use instead of `alerts` notifications custom console tab inside content example?

<img width="691" alt="image" src="https://github.com/taiga-family/taiga-
ui/assets/12021443/e4f99fe0-bc18-4320-89de-cf98104d34bf">

for example

<img width="371" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/a130c10d-fad2-47e2-b7ee-c0b91d12e000">

Reply:
in this case, it would not irritate users too much